libs/components/src/lib/Accordion/index.module.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../../../../libs/theme";
+
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  &__item {
```

Comment:
use unique class names for all items

Reply:
change container to accordion and item to accordion__item

Reply:
I mean in scss file it is not clear to define classes like this, for example for accordion__item, it's better use like this:
.accordion__item {...} not this: &__item {...}, please modify this for all classes in scss file